ree/ree.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+#ifndef REE_H
+#define REE_H
+
+
+#include "apf.h"
+#include <apfMesh.h>
+#include <PCU.h>
+#include <lionPrint.h>
+#include "apfShape.h"
+#include "apfField.h"
+#include <mthQR.h>
+#include <mth.h>
+#include <mth_def.h>
+
+namespace ree {
+
+/*
```

Comment:
@Samiullah-Malik

The descriptions (commented code before) each of the function declarations in this file has to be written in a way that is understandable to doxygen. Doxygen would then automatically create the api documentation that we have here https://www.scorec.rpi.edu/pumi/doxygen/index.html

Please look at some of the other header files in pumi (for example spr.h) and see how that is done. Let me know if you have any questions.

Reply:
I added descriptions (in a way that is understandable to doxygen) before those functions which I think should be visible in doxygen. There are a few functions which are just important for the implementation and I don't think there is a lot of utility in having them in the doxygen. Let me know if that is okay.